src/components/label-info/LabelInfo.tsx

```diff
@@ -21,7 +22,13 @@ export const LabelInfo = ({label}: LabelInfoProps) => {
 
   return (
     <View style={linkSectionItemStyle.flag}>
-      <ThemeText color="info" type="body__tertiary">
+      <ThemeText
+        // Setting color="info" uses the secondary text color, so we need to set
+        // the primary text color manually for it to work with the label
+        // background.
```

Comment:
Just remove this comment, comments clutter the code and this isn't important enough to have a comment on it IMO

Reply:
Will do!